c7n/resources/s3control.py

```diff
@@ -95,3 +97,89 @@ class resource_type(TypeInfo):
         permission_prefix = 's3'
 
     source_mapping = {'describe': MultiRegionAccessPointDescribe}
+
+
+class StorageLensDescribe(DescribeSource):
+    def get_query_params(self, query_params):
+        query_params = query_params or {}
+        query_params['AccountId'] = self.manager.config.account_id
+        return query_params
+
+    def augment(self, resources):
+        client = local_session(self.manager.session_factory).client('s3control')
+        results = []
+        for r in resources:
+            arn = Arn.parse(r['StorageLensArn'])
+            storage_lens_configuration = client \
+                .get_storage_lens_configuration(AccountId=arn.account_id, ConfigId=r['Id'])\
+                .get('StorageLensConfiguration')
+            tags = client \
```

Comment:
use `universal_augment` which can fetch bulk fetch tags for multiple resources instead of fetching per resource

Reply:
yeah, code updated using universal_augment.